scripts/run_benchmark.py

```diff
@@ -41,13 +42,19 @@ def parse_args():
 
         run_obj = {"seed": i}
         run_obj["updates"] = []
-
+        run_obj["iterations"] = []
+        # check for prefixes
         for line in update_lines:
-            time_delta, makespan = line.split()
-            run_obj["updates"].append({
+            if re.match("time: ", line):
+                newEntry = re.sub("time: ", "", line)
+                time_delta, makespan = newEntry.split()
+                run_obj["updates"].append({
                 "time_delta": int(time_delta),
                 "makespan": int(makespan)
-            })
+                })
+            if re.match("iterations: ", line):
+                newEntry = re.sub("iterations: ", "", line)
+                run_obj["iterations"].append({"iterations" : int(newEntry)})
```

Comment:
Wouldn't this be easier? Line 45 could be deleted then

```suggestion
                run_obj["iterations"] = int(newEntry)
```

Reply:
Yes, your version is simpler.